### ISSUE TITLE
Clarify whether Tabbed Carousel may have aria-roledescription

### ISSUE THREAD

**Issue by Ignacio-de-Loyola** [w3c/aria-practices#2773]
# Describe your concern
Requirements for Tabbed Carousel Elements are specified as follows.
> The structure of a tabbed carousel is the same as a basic carousel except that:
> Each slide container has role [tabpanel](https://w3c.github.io/aria/#tabpanel) in lieu of group, and it does not have the aria-roledescription property.

But, In the Tabs for Slide Control Example, it is classified as a required attribute, which way is correct?
![image](https://github.com/w3c/aria/assets/80868256/da369962-79cb-41b3-a1a9-dfd757f50c2d)

# Link to the version of the specification or documentation you were looking at at.

Link to documentation: 
Carousel Pattern - https://www.w3.org/WAI/ARIA/apg/patterns/carousel
Auto-Rotating Image Carousel with Tabs for Slide Control Example - https://www.w3.org/WAI/ARIA/apg/patterns/carousel/examples/carousel-2-tablist/

Does the issue exists in [the editors draft](https://w3c.github.io/aria/) (the editors draft is the most recent draft of the specification)?
